src/command/install.rs

```diff
@@ -47,16 +49,21 @@ Supported Tools:
             arg_version,
         }: Args,
     ) -> Fallible<Self> {
+        let version = arg_version
+            .map(VersionSpec::parse)
+            .invert()?
+            .unwrap_or_default();
+
         match &arg_tool[..] {
             "node" => {
-                Ok(Install::Node(VersionSpec::parse(&arg_version)?))
+                Ok(Install::Node(version))
             },
             "yarn" => {
-                Ok(Install::Yarn(VersionSpec::parse(&arg_version)?))
+                Ok(Install::Yarn(version))
             },
-            ref tool => Ok(Install::Other {
-                name: tool.to_string(),
-                version: VersionSpec::parse(&arg_version)?,
+            ref package => Ok(Install::Other {
+                package: package.to_string(),
+                version: version,
```

Comment:
nit: can just do `version,` here (I see frequent compiler warnings about these)

Reply:
Yeah I'm confused about when those warnings happen—for some reason I'm not seeing them on my machine.